src/HyperFormula.ts

```diff
@@ -2846,6 +2910,7 @@ export class HyperFormula implements TypedEmitter {
    * @param {SimpleCellAddress} cellAddress - cell coordinates
    *
    * @throws [[NoSheetWithIdError]] when the given sheet ID does not exist
+   * @throws [[ExpectedValueOfTypeError]] if cellAddress is of wrong type
```

Comment:
is not `[[SimpleCellAddress]]` ? @kirszenbaum WDYT, make sense to be more precise here? Probably every `ExpectedValueOfTypeError` would need a review